src/main/kotlin/uk/gov/justice/digital/hmpps/hmppstier/client/CommunityApiClient.kt

```diff
@@ -15,13 +16,30 @@ class CommunityApiClient(@Qualifier("communityWebClientAppScope") private val we
   fun getRegistrations(crn: String): Collection<Registration> {
     return webClient
       .get()
-      .uri("/offenders/crn/${crn}/registrations")
+      .uri("/offenders/crn/$crn/registrations")
       .retrieve()
       .bodyToMono(CommunityApiRegistrationsDto::class.java)
       .block()?.registrations ?: listOf()
   }
+
+  @Cacheable(value = ["deliusAssessment"], key = "{ #crn }")
+  fun getAssessments(crn: String): DeliusAssessmentsDto? {
+    return webClient
+      .get()
+      .uri("/offenders/crn/$crn/registrations")
```

Comment:
Should this not end with `/assessments` ?

Reply:
It would work better wouldn't it 👍